src/components/VolunteerLogs/YourLogsTable.jsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-key */
+import React, { useEffect, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useTable, usePagination, useSortBy } from 'react-table';
+import { Table, Tr, Td, Tbody, Spinner, Box } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Box' is defined but never used.